src/main/java/edu/berkeley/ground/GroundServer.java

```diff
@@ -14,14 +14,7 @@
 
 package edu.berkeley.ground;
 
-import edu.berkeley.ground.dao.models.EdgeFactory;
-import edu.berkeley.ground.dao.models.EdgeVersionFactory;
-import edu.berkeley.ground.dao.models.GraphFactory;
-import edu.berkeley.ground.dao.models.GraphVersionFactory;
-import edu.berkeley.ground.dao.models.NodeFactory;
-import edu.berkeley.ground.dao.models.NodeVersionFactory;
-import edu.berkeley.ground.dao.models.StructureFactory;
-import edu.berkeley.ground.dao.models.StructureVersionFactory;
+import edu.berkeley.ground.dao.models.*;
```

Comment:
We're using the [Google Java style guide](https://google.github.io/styleguide/javaguide.html) which specifies no `*` imports.